docs/prfc/drafts/pimp.xml

```diff
@@ -268,27 +260,25 @@
      </section>
      
      <section title = "Error handling">
-       <t>There are two types of errors that can occur:</t>
+       <t>There are multiple errors that can occur:</t>
        
        <t><list style="numbers">
-				 <t>Packet loss</t>
-         <t>Corrupted data (indicated by an incorrect checksum)</t>
+		        <t>Packet loss</t>
+            <t>Corrupted data (indicated by an incorrect checksum)</t>
+		        <t>*Packet with invalid flags (invalid flag combination) is received</t>
        </list></t>
        
-       <t>Both of these cases are handled the same way. A packet
+       <t>The first cases are handled the same way. A packet

```

Comment:
Seconded for handling packets with invalid flags. 

Node: We should add a minimum wait-time here (https://campuswire.com/c/G800DF8C8/feed/142)